README.md

```diff
@@ -36,12 +36,11 @@ or online at
 Documentation
 -------------
 
-If you downloaded an official release, the documentation can be found in $VIGRA_PATH/doc/vigra/, the start file 
-is $VIGRA_PATH/doc/vigra/index.html. Online documentation for the latest release is at 
-http://hci.iwr.uni-heidelberg.de/vigra/doc/vigra/index.html
+If you downloaded an official release, the documentation can be found in `$VIGRA_PATH/doc/vigra/`, the start file 
+is `$VIGRA_PATH/doc/vigra/index.html`.
 
 When you use the development version from github, you can generate documentation by `make doc`. Up-to-date 
-online documentation for the 'master' branch is at http://ukoethe.github.io/vigra/doc/vigra/index.html
+online documentation for the 'master' branch is regularly pushed to http://ukoethe.github.io/vigra/doc/vigra/
```

Comment:
Unfortunately, "is regularly pushed to" is mainly wishful thinking... I recantly read that travis can in principle generate and upload documentation automatically, but it requires to place the encrypted password into the sources. Is this safe?